0_config/src/wqp_utils.R

```diff
@@ -0,0 +1,94 @@
+inventory_wqp <- function(wqp_needs, wqp_variables) {
```

Comment:
This is tricky - 

if the number of sites is small, it is faster to just get the data than it is to re-build the inventory. If the number of sites is large, we'll need the inventory to break up the queries logically. 

Either way, I don't recommend including the logic to skip the inventory, just surprised that the query takes so long (~1.5 mins for 4 sites).